src/server.h

```diff
@@ -1040,8 +1040,9 @@ typedef struct replBacklog {
 
 typedef struct replDataBuf {
     list *blocks; /* List of replDataBufBlock */
+    size_t mem;   /* The total allocated memory in all blocks */
     size_t len;   /* Number of bytes stored in all blocks */
```

Comment:
I find the naming of `mem` and `len` a bit unclear at first glance. To improve clarity, could we revise the documentation for these members? Suggestion:

```suggestion
    size_t mem;  /* Total allocated memory including buffer metadata and list nodes */
    size_t len;  /* Total replication data bytes pending processing */
```